### PR TITLE
add-byod-policy

### PR DESCRIPTION
**Pull request by robmarcer**
Fun fact, this was 100% written by a well known chat bot with conversational prompts from me. I didn't edit a single character.

### CODE REVIEW

File: peopleops/index.md [hunk 1]
```diff
@@ -133,3 +133,26 @@ For example:
...
+- Employees are responsible for backing up their own data on their personal devices. FlowForge will not be held responsible for any data loss or damage to personal devices.
+- Personal devices must be fit for purpose, meaning that they must have the necessary hardware and software capabilities to enable the employee to perform their job duties.
+- Employees must not use their personal devices to store or access any confidential or sensitive company information. If necessary, employees may be required to use a company-issued device or a secure virtual environment to access such information.
+
+#### Budget for Personal Devices
+
+Employees will be provided with a monthly budget of $100 to cover the cost of their personal devices, such as laptops and monitors. This budget is only available if the employee buys all of their own equipment. The budget can be used to cover the cost of purchasing new equipment, as well as any ongoing expenses, such as maintenance and repair costs.
```
Comment (ZJvandeWeg) [inline peopleops/index.md:154]: This is the only line I don't quite understand, how does that work when my headphones are 200$?

Comment (ZJvandeWeg) [reply peopleops/index.md:154]: Should it be a yearly budget of 1200? And how does that work if someone want to transition to non-BYOD?

Comment (robmarcer) [reply peopleops/index.md:154]: > This is the only line I don't quite understand, how does that work when my headphones are 200$?

The idea is that the team-member gets the $100 every month regardless of if they need to buy something or not. The expectation is that $100 a month should be enough to over time cover the depreciation of the computer equipment needed to do the job so by the time they need to buy something they should have the money to cover it. This works in a similar way to how company car allowances work - https://www.driversnote.com/dictionary/car-allowance

Comment (robmarcer) [reply peopleops/index.md:154]: > Should it be a yearly budget of 1200? And how does that work if someone want to transition to non-BYOD?

I suspect quarterly might be a good middle ground if trying to keep the admin down. As far as I can see, the transition from BYOD is simple, the individual would at that point purchase and expenses the equipment they need as per the handbook and the BYOD payments would end.

**Pull request closed without merging**